content/en/docs/reference/identifiers/identifiers.md

```diff
@@ -0,0 +1,109 @@
+# Identifiers and Names in Kubernetes
```

Comment:
We should add [front matter](https://gohugo.io/content-management/front-matter/) here and _not_ set a title using a level-1 Markdown heading.

@imoisharma have a look at other reference documents for an example.

Reply:
Sorry about this, as it is my first time with this things. I have added the front-matter. Have a look :)  @sftim